libs/coin-modules/coin-multiversx/src/buildTransaction.ts

```diff
@@ -33,11 +33,56 @@ export const buildTransactionToSign = async (
     sender: address,
     gasPrice: GAS_PRICE,
     gasLimit: transaction.gasLimit || networkConfig.MinGasLimit.valueOf(),
-    data: transaction.data,
     chainID,
     ...HASH_TRANSACTION,
   };
 
+  if (transaction.data) {
+    unsigned.data = transaction.data;
+  }
+
   // Will likely be a call to Elrond SDK
   return JSON.stringify(unsigned);
 };
+
+export const serializeESDTInfo = (
+  ticker: string,
+  id: string,
+  decimals: number,
+  chainId: string,
+  signature: string,
+): Buffer => {
+  const tickerLengthBuffer = Buffer.from([ticker.length]);
```

Comment:
Should not be an issue in practice, but `Buffer.from([somethingLargerThan255])` does not work properly.

Reply:
the ticker can't be so long (protocol constraint) and I did not modify this code, just extracted it. I am assuming this worked previously.